.github/workflows/testing.yml

```diff
@@ -105,7 +105,12 @@ jobs:
       # (where we run a reduced set of tests).
       - name: Run tests
         if: matrix.type == 'tests'
+        env:
+          VET_BUILD_GO_PROTOC: 1
         run: |
+          unset VET_SKIP_PROTO
```

Comment:
Maybe we should just set `VET_SKIP_PROTO` for the "Run vet.sh" case and not set it at all for "Run tests"

Reply:
@arvindbr8 sounds good, will update